api/ssh.go

```diff
@@ -337,7 +337,7 @@ func SSHSign(w http.ResponseWriter, r *http.Request) {
 		}
 		identityCertificate = certChainToPEM(certChain)
 	}
-
+	LogSSHCertificate(w, cert)
```

Comment:
We should log on renew too.

Reply:
Added in latest commit for renew and rekey